packages/roosterjs-content-model/lib/editor/corePlugins/ContentModelCopyPastePlugin.ts

```diff
@@ -83,72 +83,71 @@ export default class ContentModelCopyPastePlugin implements PluginWithState<Copy
     }
 
     private onCutCopy(event: Event, isCut: boolean) {
-        if (this.editor) {
-            const selection = this.editor.getSelectionRangeEx();
-            if (selection && !selection.areAllCollapsed) {
-                const model = this.editor.createContentModel({
-                    disableCacheElement: true,
+        if (!this.editor) {
+            return;
+        }
+        const selection = this.editor.getSelectionRangeEx();
+        if (selection && !selection.areAllCollapsed) {
+            const model = this.editor.createContentModel({
+                disableCacheElement: true,
+                cloneEntityElement: true,
```

Comment:
Question: Is it ok to always clone entity element if disableCacheEelemnt (line 92) is true? If that is ok, we don't need to introduce this option

Reply:
Since we only enable disableCacheElement & this new option for copy/paste, I think it make sense to do that.
Removing this new option